webapp/pages/logout.vue

```diff
@@ -36,8 +36,17 @@
 </template>
 
 <script>
+import HcImage from '~/components/Image'
 export default {
+  components: {
+    HcImage
+  },
   layout: 'blank',
+  computed: {
+    imageProps() {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-09T11:21:10Z" title="Thursday, May 9th 2019, 1:21:10 pm +02:00">May 9, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -36,8 +36,17 @@
 </template>
 
 <script>
+import HcImage from '~/components/Image'
 export default {
+  components: {
+    HcImage
+  },
   layout: 'blank',
+  computed: {
+    imageProps() {
```


@aonomike this method got overlooked? Or is there a reason for keeping it in.